json-definitions/v3/tech-record/get/trl/complete/index.json

```diff
@@ -478,40 +478,35 @@
     },
     "techRecord_couplingCenterToRearAxleMax": {
       "type": [
-        "integer",
-        "null"
+        "integer"
```

Comment:
need to remove these from arrays to be just "type": "integer" for example if only one thing